yarn.lock

```diff
@@ -5128,6 +5183,11 @@ is-glob@^4.0.0, is-glob@^4.0.1, is-glob@~4.0.1:
   dependencies:
     is-extglob "^2.1.1"
 
+is-map@^2.0.1:
```

Comment:
all of these changes are not related to mocha, make sure you just test what the PR had